sycl/plugins/unified_runtime/pi2ur.cpp

```diff
@@ -8,34 +8,3 @@
 
 // This thin layer performs conversion from PI API to Unified Runtime API
 // TODO: remove when SYCL RT is changed to talk in UR directly
-
-#include <pi2ur.hpp>
-#include <ur.hpp>
-
-// Early exits on any error
-#define HANDLE_ERRORS(urCall)                                                  \
-  if (auto Result = urCall)                                                    \
-    return ur2piResult(Result);
-
-__SYCL_EXPORT pi_result piPlatformsGet(pi_uint32 num_entries,

```

Comment:
Could you please clarify why these functions were moved to header file?

Reply:
they are very thin and I didn't want extra call overhead so made them inline